src/main/java/Deadlines.java

```diff
@@ -0,0 +1,28 @@
+public class Deadlines extends Task {
+    private final String DEAD_LINE_SYMBOL = "[D]";
```

Comment:
*Very* small comment: should probably be DEADLINE_SYMBOL instead of DEAD_LINE_SYMBOL

Reply:
Agreed. I would change the name of the constant.